BiasAndFairnessModule/tests/test_metrics.py

```
@@ -0,0 +1,374 @@
+
+
+def test_regression_metrics(y_true, y_pred, y_scores, sensitive_attributes):
+    """Test regression metrics."""
+    print("Testing regression metrics...")
+    
+    results = {}
+    
+    try:
+        results["regression_demographic_parity"] = regression_demographic_parity(y_true, y_pred, sensitive_attributes)
+        print(f"  ✅ regression_demographic_parity: {results['regression_demographic_parity']:.4f}")
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)